.travis.yml

```
@@ -83,6 +83,6 @@ jobs:
       after_success: skip
 
 after_success:
-  - julia -e 'if Sys.islinux() && string(VERSION)[1:3] == "1.4"
+  - julia -e 'if Sys.islinux() && string(VERSION)[1:3] == "1.5"
```

```suggestion
  - julia -e 'if Sys.islinux()
```
There is no harm in building every linux, so we can just avoid deciding which release.